action.yml

```diff
@@ -17,25 +17,35 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Download latest kube-linter
+    - name: Download the latest kube-linter
```

Comment:
Another general thought: we should probably allow people to specify the `kube-linter` version optionally.

Reply:
Should be possible but I don't want to invest into this now. Getting SARIF output to work might be more beneficial for users.

Reply:
SG.